check.go

```diff
@@ -190,6 +198,31 @@ func init() {
 		return nil
 	})
 
+	path = "epp > response > extension > " + ExtFee06 + " chkData"
```

Comment:
Yeah, the XML structure is just slightly different, so I set up the scanner to handle whichever is present